web/src/settings_streams.js

```diff
@@ -36,7 +36,7 @@ function get_chosen_default_streams() {
 
 function create_choice_row() {
     const $container = $("#default-stream-choices");
-    const value = settings_profile_fields.get_value_for_new_option("#default-stream-choices");
+    const value = settings_profile_fields.get_value_for_new_option($("#default-stream-choices"));
```

Comment:
You can just pass the `$container` variable declared above, instead of using `$` to convert the string to jQuery object.